apps/web/app/layouts/nav/nav-item.layout.tsx

```diff
@@ -32,12 +35,26 @@ export function NavItem({ children, icon, href }: NavItemProps) {
       _hover={{ bg: 'ink.component-background-hover' }}
       _focusVisible={{ textDecoration: 'underline' }}
     >
-      {children}
+      <Box display="flex" alignItems="center" justifyContent="space-between" gap="space.02">
+        {children}
+        {newTab && (
+          <Box
+            width="16px"
+            height="16px"
+            alignItems="center"
+            justifyContent="center"
+            mr="space.04"
+            className={css({ display: 'none', _groupHover: { display: 'flex' } })}
```

Comment:
Do you need a `className` for that other styling?

Reply:
+1, these should be added as style props. https://panda-css.com/docs/concepts/style-props